zedc/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Recent Changes
+
+- **Enhancement:** Added `zedc status` command to display working tree status, diagnostics and system info in a developer's environment.
+- **Enhancement:** Added `zedc test coverage` command to show patch coverage and uncovered lines with hyperlinks (for [compatible terminals only](https://github.com/Alhadis/OSC8-Adoption))
```

Comment:
Generally speaking, can we add like a progress bar for the `zedc test` command? 

It's be kind of cool to get a sense of how many tests are left 😋 

Also... should `--verbose` show all tests as they run? 🤔